Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -0,0 +1,348 @@
+                        <ContentPresenter />
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style TargetType="controls:Expander">
+        <Setter Property="ExpanderToggleButtonStyle" Value="{StaticResource HeaderToggleButtonStyle}" />
+        <Setter Property="HeaderButtonStyle" Value="{StaticResource HeaderButtonStyle}" />
+        <Setter Property="HeaderButtonContent" Value="Header" />    
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:Expander">
+                    <Grid d:DesignWidth="600" d:DesignHeight="400">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition />
+                        </Grid.RowDefinitions>
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="CommonStates">
+                                <VisualState x:Name="Normal" />
+                                <VisualState x:Name="ReadOnly" />
+                                <VisualState x:Name="MouseOver" />
+                                <VisualState x:Name="Disabled" />
+                                <VisualState x:Name="Enabled" />
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="HeaderStates">
+                                <VisualState x:Name="HeaderExpanderMouseOver" />
+                                <VisualState x:Name="HeaderSelectorMouseOver" />
+                                <VisualState x:Name="HeaderExpanderPressed" />
+                                <VisualState x:Name="HeaderSelectorPressed" />
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="ContentStates">
+                                <VisualState x:Name="ContentExpanded" />
+                                <VisualState x:Name="ContentCollapsed" />
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+                        <Grid Grid.RowSpan="2" VerticalAlignment="Top">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" x:Name="PART_MainContentRow" />
+                            </Grid.RowDefinitions>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition />
+                            </Grid.ColumnDefinitions>
+                            <ToggleButton x:Name="PART_ExpanderToggleButton" Height="40"
+                                          Style="{TemplateBinding ExpanderToggleButtonStyle}" VerticalAlignment="Top"
```

Comment:
No need for a ToggleButton and a Button, just one ToggleButton styled to have the arrow and the content of the header